templates/snmpd/snmpd.conf.erb

```diff
@@ -5,23 +5,23 @@
 ################################################################################
 # AGENT BEHAVIOUR
 
-<% if @master != nil -%>
+<% if @master.nil? -%>
 <% if @master -%>
 master agentx
 
-<% if @agentx_perms != nil -%>
+<% if @agentx_perms.nil? -%>
```

Comment:
This template is used by snmp module.  I just removed some things from it that I put other places.